examples/src/components/BucketExecutorExample.jsx

```diff
@@ -0,0 +1,250 @@
+// (C) 2007-2019 GoodData Corporation
+/* eslint-disable react/jsx-closing-tag-location */
+import React, { Component } from "react";
+import { BucketExecutor, LoadingComponent, ErrorComponent, Model } from "@gooddata/react-components";
+
+import { totalSalesIdentifier, projectId, locationStateDisplayFormIdentifier } from "../utils/fixtures";
+
+const presets = {
+    kpi: {
+        props: {
+            dimensions: [[Model.measure(totalSalesIdentifier)]],
+        },
+        label: "KPI",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result }) => {
+            return (
+                <p
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        height: 60,
+                        margin: "10px 0",
+                        fontSize: 50,
+                        lineHeight: "60px",
+                        whiteSpace: "nowrap",
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.data[0]}
+                </p>
+            );
+        },
+    },
+    attributeValueList: {
+        props: {
+            dimensions: [[Model.attribute(locationStateDisplayFormIdentifier)]],
+        },
+        label: "List of States",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result }) => {
+            return (
+                <p
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        minHeight: 60,
+                        margin: "10px 0",
+                        fontSize: 50,
+                        lineHeight: "60px",
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.headerItems[0][0].map(header => header.attributeHeaderItem.name).join(", ")}
+                </p>
+            );
+        },
+    },
+    pagedExecution: {
+        props: {
+            dimensions: [
+                [Model.attribute(locationStateDisplayFormIdentifier)],
+                [Model.measure(totalSalesIdentifier)],
+            ],
+            initialPaging: {
+                offset: [0, 0],
+                limit: [1, 1],
+            },
+        },
+        label: "Paged execution",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result, getPage }) => {
+            const isLastPage = result.paging.offset[0] + result.paging.count[0] >= result.paging.total[0];
```

Comment:
Helpers for these four things could be useful in other places as well :)

Reply:
This is too opinionated